### PR TITLE
Fix Panache+Flyway tests in native

### PR DESCRIPTION
**Pull request by Sgitario**

### CODE REVIEW

**Comment by Sgitario** [comment]
Fixes https://github.com/quarkus-qe/beefy-scenarios/issues/47

File: 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/PostgreSqlApplicationResourceTest.java [hunk 1]
```diff
@@ -30,10 +29,9 @@
     private ApplicationEntity actualEntity;
     private ApplicationEntity[] actualList;
 
-    @BeforeEach
-    @Transactional
+    @AfterEach
     public void setup() {
```
Comment (rsvoboda) [inline 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/PostgreSqlApplicationResourceTest.java:33]: Why not just
```java
    @AfterEach
    public void deleteEntityIfExists() {
        if (actualEntity != null) {
            whenDeleteApplication();
        }
    }
```
? (and `setup` as method name for `@AfterEach` doesn't feel right)

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/PostgreSqlApplicationResourceTest.java (hunk 1 above)
Comment (Sgitario) [reply 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/PostgreSqlApplicationResourceTest.java:33]: Sorry, I tried different things and this was a leftover. Changed it to tearDown.

**Review by rsvoboda — APPROVED** [review]

**Comment by rsvoboda** [comment]
`Fixes #47` needs to be specified in the description to do auto close of the issue, closing the issue manually now.

**Pull request merged**